Ported/dots-training-2022-03-na-group1_BucketBrigade/Assets/Scripts/Components/BucketFill.cs

```diff
@@ -0,0 +1,12 @@
+using Unity.Entities;
+
+/// <summary>
+///     Holds the cooldown time for a bot filling a bucket.
+/// </summary>
+struct BucketFill : IComponentData
+{
+    /// <summary>
+    ///     The time the filling process is finished.
+    /// </summary>
+    public float cooldown;
```

Comment:
been following the practice of single value components having Value as their field.